build/vega-lite-schema.json

```diff
@@ -4990,6 +5010,16 @@
                 }
             },
             "type": "object"
+        },
+        "{ color?: NonspatialResolve; opacity?: NonspatialResolve; size?: NonspatialResolve; shape?: Nonsp...": {
```

Comment:
Yeah, I looked into fixing this properly with the ts -> json converter. I think we can do it but I need some more time. 

@YuhanLu can you add the missing sed calls?